src/components/Detail/Detail.jsx

```diff
@@ -1,8 +1,17 @@
 import {Bookmark, Close, Delete} from '@mui/icons-material';
+import {NavLink} from 'react-router-dom';
+
+import {useStep, useCreate} from '../../hooks/useForm';
 
 import DetailPOP from './Detail.styled';
 
 const Detail = ({event, back, deleteEvent, bookmark}) => {
```

Comment:
Don't you pass those properties down where you call the Details component in App.js?

Reply:
The App.js route is currently no longer used at all. That was just my first thought this morning. I have removed the route there again